README.md

```diff
@@ -595,6 +596,95 @@ Output:
+
+`prx-wavefile` will attempt to set the `fmt`, `mext`, `bext`, `fact`, and `data` chunks from the source file, e.g. analyzing the mpeg audio to set the `fmt` sample rate, bit rate, and number of channels.
+
+The `cart` chunk is optional, and won't be set unless it's included in the `Task.Chunks` array, as in the example below.
+
+The Output includes a `WavfileChunks` array with the attributes set on the `prx-wavefile` chunks.
+
+Input:
+
+```json
+{
+    "Type": "WavWrap",
+    "Destination": {
+        "Mode": "AWS/S3",
+        "BucketName": "myBucket",
+        "ObjectKey": "myTranscript.json"
+    },
+    "Chunks": [
+        {
+            "ChunkId": "cart",
+            "Version": "0101",
+            "CutId": "12345",
+            "Title": "Title",
+            "Artist": "Artist",
+            "StartDate": "2020/01/01",
+            "StartTime": "00:00:00",
+            "EndDate": "2020/01/14",
+            "EndTime": "00:00:00",
+            "ProducerAppId": "PRX",
+            "ProducerAppVersion": "3.0"
+        }
+    ]
+}
+```
+
+Output:
+
+```json
+{
+    "Task": "WavWrap",
+    "Mode": "AWS/S3",
+    "BucketName": "myBucket",
+    "ObjectKey": "myTranscript.json",
+    "Time": "2012-12-21T12:34:56Z",
+    "Timestamp": 1356093296.123,
+    "WavefileChunks": [
+        {
+            "chunkId": "cart",
+            "chunkSize": 2048,
+            "version": "0101",
+            "title": "Title",
+            "artist": "Artist",
+            "cutId": "12345",
+            "clientId": "",
+            "category": "",
+            "classification": "",
+            "outCue": "",
+            "startDate": "2020/01/01",
+            "startTime": "00:00:00",
+            "endDate": "2020/01/14",
+            "endTime": "10:00:00",
+            "producerAppId": "PRX",
+            "producerAppVersion": "3.0",
+            "userDef": "",
+            "levelReference": 0,
+            "postTimer": [],
+            "reserved": "",
+            "url": "",
+            "tagText": ""
+        }
+    ]
+}
+```
+
+#### AWS/S3
+
+S3 destinations are done by the [upload()](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#upload-property) method in the AWS Node SDK.
+
+The `BucketName` and `ObjectKey` properties are required.
+
+To set metadata on the new audio file object, use the optional `Parameters` property on the destination. The contents of `Parameters` are passed directly to the [upload()](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#upload-property) method.
+
+If you set the optional `ContentType` property to `REPLACE`, the content type of the newly created audio file will be set to a [heuristically-determined](https://www.npmjs.com/package/file-type) value from the job's source file. If the content type could not be determined heuristically, this property has no effect. If a `ContentType` value is explicitly defined in `Parameters` that value will take precedence.
```

Comment:
I wonder if we just remove this ability for this task. Do you think there'd ever be a reason to use it for this operation?